frontend/src/tests/test_data.js

```diff
@@ -0,0 +1,258 @@
+import instructorsJSON from "./samples/instructors.json";
```

Comment:
`test_data.js` should not be in the test's directory. It should be in a subfolder where we can keep all the data-related files together.